### ISSUE TITLE
Brightness of displays

### ISSUE THREAD

**Issue by froggi1930** [flybywiresim/a32nx#5451]
### Aircraft Version

Development

### Description

Is it possible to set the brightness to a different default value, so that they start much brigher when loading the plane?

### Discord Username (optional)

Georg Stunić#6211

**Comment by wpine215** [comment]
I agree - having them all at slightly different values between 50 - 70% would be ideal.

**Comment by MisterChocker** [comment]
The reason the display brightness is that low when loading in at the gate is to save display cycles, this is also done in real life.

@St54Kevin

**Comment by St54Kevin** [comment]
> The reason the display brightness is that low when loading in at the gate is to save display cycles, this is also done in real life.
> 
> @St54Kevin

Yes I can confirm that. When leaving the plane I turn them down to the lowest setting. The thing is we dont have persistence yet between sessions, which would make things a little easier because people could just leave it as it is. 
But I for myself also turn them up to 100% in daylight everytime I load the plane. So I actualy dont know what we wanna do. I agree it would be user friendlier with higher brightness on daytimes

**Comment by MisterChocker** [comment]
I'd suggest make the ac remember last brightness setting. (maybe make it an option on the efb..)

**Comment by LordNecton** [comment]
The  FMC seems really dull also in VR at max default brightness

**Comment by davidwalschots** [comment]
> I'd suggest make the ac remember last brightness setting. (maybe make it an option on the efb..)

I'd suggest solving the problem of aircraft state persistence in one go. Otherwise the next issue will be "Remember button _x_." 😀

**Comment by ErickSharp** [comment]
> I'd suggest make the ac remember last brightness setting. (maybe make it an option on the efb..)

This is very feasible but it will have to be on hold until me and some other people can get a tabbed settings page for the efb up and running.

**Comment by tracernz** [comment]
My suggestion is an EFB option to link all the display knobs so it's less tedious to change them all if you like. We should do the same for altimeter knobs (and split the captain/fo side at the same time).

**Comment by St54Kevin** [comment]
I just noticed we have different brightness loading states for different aircraft states (very low on C&D and 12 oclock position inflight/RW). I suggest putting the inflight loading state to 100%. If possible 100% on daytime and 50% on night time. @tracernz @davidwalschots

**Comment by bouveng** [comment]
> I'd suggest solving the problem of aircraft state persistence in one go. Otherwise the next issue will be "Remember button _x_." 😀

I have many ideas regards to this as saving states for buttons, knobs, switches and dials is on my personal wishlist.

**Comment by froggi1930** [comment]
> I just noticed we currently have different brightness loading states for different aircraft states anyway (very low on C&D and 12 oclock position inflight/RW). I suggest putting the inflight loading state to 100%. If possible 100% on daytime and 50% on night time. @tracernz @davidwalschots
> This also includes FCU brightness

Is it possible to modify the setting file myself in the meantime? If so where can I find the settings?

**Comment by bouveng** [comment]
> Is it possible to modify the setting file myself in the meantime? If so where can I find the settings?

Sure you can, look in the *.FLT files.

Br.

**Comment by P1NBA11ER** [comment]
I would suggest an option via the EFB to have preferred integral, cockpit and display lighting percentages stored. So once set to your liking, they stayed there. I often fly at night so I tend to like the displays a bit dimmer but the integrals a bit brighter.

**Comment by SagiEran** [comment]
> I would suggest an option via the EFB to have preferred integral, cockpit and display lighting percentages stored. So once set to your liking, they stayed there. I often fly at night so I tend to like the displays a bit dimmer but the integrals a bit brighter.

Maybe an option to capture current brightness settings of the knobs, much like the Throttle calibration option. Once "Captured" at the EFB it will save the settings in the configuration file for future use.

**Comment by derl30n** [comment]
Brightness presets https://github.com/flybywiresim/a32nx/issues/6230 will be added by https://github.com/flybywiresim/a32nx/pull/6528

**Comment by froggi1930** [comment]
thanks a lot for this addition
